omnibus/config/software/datadog-iot-agent.rb

```diff
@@ -113,6 +113,17 @@
       copy 'bin/trace-agent/trace-agent.exe', "#{Omnibus::Config.source_dir()}/datadog-iot-agent/src/github.com/DataDog/datadog-agent/bin/agent"
     end
   end
+  block do
```

Comment:
Do we want the security check in the IOT agent package ? Are we sure we don't use any CGO in the security agent ?

Reply:
There's no cgo in the security agent at this point. It's included in the IoT build now, we could take it out if there are strong reasons to do so, but generally it should probably be included in all targets?

Reply:
:+1: